src/services/completions.ts

```diff
@@ -637,7 +638,7 @@ namespace ts.Completions {
             if (type) {
                 // Filter private properties
                 for (const symbol of type.getApparentProperties()) {
-                    if (typeChecker.isValidPropertyAccess(<PropertyAccessExpression>(node.parent), symbol.name)) {
+                    if (typeChecker.isValidPropertyAccess(<PropertyAccessExpression>(node.parent), symbol.getName())) {

```

Comment:
I'm not sure where people are using `symbol.name` from the outside, as opposed to `symbolToString`. But I would expect a method `getName()` to just get the `name` property, or else be named something else.